app/controllers/application_controller.rb

```diff
@@ -3,6 +3,11 @@ class ApplicationController < ActionController::Base
 
   include RealIpLogger
 
+  # Error modal contact message
+  ZENDESK_CONTACT = "If this error persists and you require assistance, please contact support at " \
+                    "<a href='mailto:scp-support@broadinstitute.zendesk.com' id='zendesk-alert-email'>" \
+                    "scp-support@broadinstitute.zendesk.com</a>."
+
```

Comment:
I think we should add a whitespace: no-wrap  css class to this so that the email address never line-wraps

Reply:
Also, a few line breaks between the error message and support email message might help.

Reply:
Can do.

Reply:
390a54e